shared/chat/conversation/info-panel/notifications/container.js

```diff
@@ -4,10 +4,33 @@ import logger from '../../../../logger'
 import * as Constants from '../../../../constants/chat'
 import * as Types from '../../../../constants/types/chat'
 import * as ChatGen from '../../../../actions/chat-gen'
-import {Notifications, type Props} from '.'
-import {connect, type TypedState} from '../../../../util/container'
+import {Notifications} from '.'
+import {compose, connect, type TypedState, lifecycle, type Dispatch} from '../../../../util/container'
 import {type DeviceType} from '../../../../constants/types/devices'
 
+type StateProps = {
+  channelWide: boolean,
+  desktop: Types.NotifyType,
+  mobile: Types.NotifyType,
+  muted: boolean,
+  saveState: Types.NotificationSaveState,
+}
+
+type DispatchProps = {
+  _resetSaveState: (conversationIDKey: Types.ConversationIDKey) => ?Promise<*>,
```

Comment:
You typed this as returning `any`, but why not an inferred/more specific type like `*` or `Promise<*>`?

Reply:
ideally we'd never return promises from our dispatches. that should be handled by the redux store. you're not even using the return value at all. really we should treat it as void

Reply:
Okay, now I'm confused -- it was originally `void`, then your PR changed it to `any`. So it should be `void` after all?

Reply:
Per chat, changed back to void and put braces